doc/api/fs.md

```diff
@@ -574,6 +574,8 @@ changes:
     description: The `path` parameter can be a WHATWG `URL` object using `file:`
                  protocol. Support is currently still *experimental*.
 -->
+Warning: this is **blocking call**, do not use in an asynchronous environment unless
```

Comment:
Hi, @JonathanDn! Welcome and thanks for this! The text seems like it can stand some improvements. Here are my suggestions.

* Please remove the comma splice.
* I'd also remove the typographic emphases (bold text, starting with a "Warning: " label), as we overuse those in our documents to an almost comical degree.
* Avoid link text of "here". Indicate what it is.

All told, something like this?:

```md
Note that this is a blocking call. Avoid this method in an asynchronous
environment. See
[Overview of Blocking vs Non-Blocking](https://nodejs.org/en/docs/guides/blocking-vs-non-blocking)
for more information.
```

Reply:
Thanks a whole lot for the feedback Trott ! :+1: